test/functional/http/get/accounts.js

```diff
@@ -14,226 +15,212 @@ describe('GET /api/accounts', function () {
 
 		it('using known address should be ok', function () {
 			return getAccountsPromise('address=' + node.gAccount.address).then(function (res) {
-				node.expect(res).to.have.property('success').to.be.ok;
-				node.expect(res).to.have.property('account').that.is.an('object');
-				node.expect(res.account).to.have.property('address').to.equal(node.gAccount.address);
-				node.expect(res.account).to.have.property('unconfirmedBalance').that.is.a('string');
-				node.expect(res.account).to.have.property('balance').that.is.a('string');
-				node.expect(res.account).to.have.property('publicKey').to.equal(node.gAccount.publicKey);
-				node.expect(res.account).to.have.property('unconfirmedSignature').to.equal(0);
-				node.expect(res.account).to.have.property('secondSignature').to.equal(0);
-				node.expect(res.account).to.have.property('secondPublicKey').to.equal(null);
-				node.expect(res.account).to.have.property('multisignatures').to.a('array');
-				node.expect(res.account).to.have.property('u_multisignatures').to.a('array');
+				node.expect(res).to.have.property('status').to.equal(200);
```

Comment:
Please extract it as a separate function `expectNonEmptyAccount` and reuse where necessary